README.md

```diff
@@ -49,48 +49,46 @@ at [getcomposer.org](http://getcomposer.org/doc/00-intro.md#installation-nix).
 You can then install this project template using the following command:
 
 ~~~
-composer create-project --prefer-dist --stability=dev yiisoft/yii-project-template myapp
-composer require yiisoft/yii-base-web
-cd myapp
-./vendor/bin/yii serve
+$ composer create-project --prefer-dist --stability=dev yiisoft/yii-project-template myapp
+$ cd myapp
 ~~~
 
-Now you should be able to access the application via `http://localhost:8080/`.
-
-### Docker
-
-Clone the repository and create the environment configuration file
-
-    cp .env.dist .env
-
 > Depending on your host system you might need to give write permissions to `./runtime` and `./public/assets`
 
+### Web application
+
 Choose an asset distribution system, e.g. if you plan to use a web-application base
```

Comment:
this is fine if you are used to working with Yii, but "Choose an asset distribution system" is not a suitable task if someone wants to try it out, we should provide a default system that is installed, which can be replaced with something else if desired.

Reply:
You can copy & paste just this - or web create a script `create-web-application-with-packagist.sh`

```
APP_DIR=_app3

composer create-project --prefer-dist --stability=dev yiisoft/yii-project-template $APP_DIR && \
cd $APP_DIR && \
composer require "wikimedia/composer-merge-plugin" && \
composer config repositories.ap '{"type": "composer", "url": "https://asset-packagist.org"}' && \
composer config extra.merge-plugin.include "vendor/*/*/composer.assets.json" && \
composer require yiisoft/yii-base-web
```

---

But in the end a developer should be aware of this setup.

Reply:
Looks very complicated.

Reply:
Anyway, let's have it in "quick start" section.

Reply:
It "looks" complicated, because there's some logic and an external web-service behind it.

*Just thinking loud ...* `curl -sL https://get.yiiframework.com/create_web_project | bash -`

This one looks less complicated:

```
composer create-project yiisoft/yii-project-template . && \
composer require foxy/foxy && composer require yiisoft/yii-base-web
```

Reply:
Yeah, we will be able to make that simpler.